qa/tasks/scripts/ses_disk_fault_injection.sh

```diff
@@ -10,7 +10,7 @@ function health_ok() {
 }
 
 storage_device_name=$(salt $disk_storage_minion cmd.run \
- "pvdisplay | grep -B 1 'VG Name .* ceph' | egrep -v 'ceph-block-dbs|nvme|--' | head -1 | cut -d / -f 3" --output=json | jq -r .[])
+ "pvdisplay 2>/dev/null | grep -B 1 'VG Name .* ceph' | egrep -v 'ceph-block-dbs|nvme|--' | head -1 | cut -d / -f 3" --output=json | jq -r .[])
```

Comment:
Why do we need to hide the stderr here and after?

Reply:
Is this because cmd.run converts stderr to stdout?

Reply:
This is tricky because we will not see any error message in case of the real issue.

Reply:
I think it gets bogus with the json output of cmd.run

Reply:
instead of having the script use salt to run a bunch of commands on another node in order to get the "storage_device_name", would it make more sense to do this in python and pass the storage_device_name as an argument to the script?

Reply:
@smithfarm why not to convert this script to the python code of the tasks it is using?

Reply:
I've reworked the multi pipe areas and removed pvdisplay usage from these scripts